flow/Histogram.cpp

```diff
@@ -100,7 +100,8 @@ const char* const Histogram::UnitToStringMapper[] = { "microseconds",
 	                                                  "bytes",
 	                                                  "bytes_per_second",
 	                                                  "percentage",
-	                                                  "count" };
+	                                                  "count", 
+													  "none"};
```

Comment:
I think the code wasn't clang formatted, there's something weird on this line with spacing.